juniper_official/Chassis/check-issu-failure.rule

```diff
@@ -0,0 +1,56 @@
+/*
+ * ISSU (In-Service Software Upgrade) failure detection
+ *
+ * If ISSU is triggered and it fails, this rule will detect the failure.
+ * It will also report during which phase the ISSU failed (validation, slave upgrade etc.).
+ */
+iceberg {
+    topic chassis.issu {
+        rule check-issu-failure {
+            synopsis "Detect ISSU failure";
+            description "This rule detects ISSU failure and displays the ISSU stage when it failed";
+            sensor issu-sensor {
+                open-config {
+                    sensor-name /junos/chassis/issu;
+                    frequency 60m;
+                }
+            }
+            field current-issu-stage {
+                sensor issu-sensor {
+                    path /junos/chassis/issu/current-issu-stage;
+                }
+                type string;
+                description "Current ISSU state";
+            }
+            field failure-stage {
+                sensor issu-sensor {
+                    path /junos/chassis/issu/failure-stage;
+                }
+                type string;
+                description "ISSU failure state";
+            }
+            trigger report-issu-failure {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";